src/app/review-results/dul-review-results.controller.js

```diff
@@ -67,12 +67,22 @@
             $scope.election.finalRationale = '';
         }
         $rootScope.path = 'dul-review-results';
-        $scope.dul = electionReview.consent.dataUseLetter;
+
+        // if both values, dulName and StructuredDataUseLetter, are missing in election,
+        // then consent's correct values will be displayed
+        if (cmElectionService.hasElectionValues($scope.election.dulName, $scope.election.translatedUseRestriction)) {
+            $scope.dul = $scope.election.dataUseLetter;
+            $scope.dulName = $scope.election.dulName;
+            $scope.structuredDataUseLetter = $sce.trustAsHtml($scope.election.translatedUseRestriction);
+        } else {
+            $scope.dul = electionReview.consent.dataUseLetter;
+            $scope.dulName = electionReview.consent.dulName;
+            $scope.structuredDataUseLetter = $sce.trustAsHtml(electionReview.consent.translatedUseRestriction);
+        }
```

Comment:
So much duplication in the controllers 😦 I don't know if there's a better way to do this, or even if it's worth it at this stage. If there is a potential overhaul to the UI framework, then I think it's fine to ignore this for now.

Reply:
That's fair, i'll close both PRs from ui and backend we will handle this differently by making a script to update the data base like Andrea suggested in [Ui PR-274](https://github.com/DataBiosphere/consent/pull/274). By doing that these changes wouldn't be needed.